src/SyncTrayzor/Pages/AboutView.xaml

```diff
@@ -6,6 +6,7 @@
         Height="400" Width="250"
         WindowStyle="None" SizeToContent="Height"
         ShowInTaskbar="False"
+        ResizeMode="CanMinimize"
```

Comment:
Given that WindowStyle is None, and ShowInTaskbar is False, is there much point in using CanMinimize over NoResize?